src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java

```diff
@@ -50,6 +49,9 @@ public FinancialTransactionDTO createFinancialTransaction(@Valid FinancialTransa
 
     @Override
     public List<FinancialTransactionDTO> getFinancialTransactionsByWalletId(@Min(1) @NotNull Long walletId) {
+        if (!walletRepository.existsById(walletId)){
```

Comment:
How it should be formatted:
`if (!walletRepository.existsById(walletId)){`
or
`if (!walletRepository.existsById(walletId)) {`
?

Reply:
You'd have to ask the author of the method